deploy/olm-catalog/bundle/manifests/migrationcontroller.crd.yaml

```diff
@@ -1,19 +1,50 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: migrationcontrollers.migration.openshift.io
 spec:
+  conversion:
+    strategy: None
```

Comment:
If we don't have different versions do we have to set this?

Or is it needed during upgrade case?

Reply:
To be honest I'm not sure. My process here was to use a known working generated v1 CRD (e.g. MigCluster) and strip down the fields until it played nicely with the APIserver and OLM.

We could try removing it if you think it will pose a problem.